OmniSharpSublime.sublime-settings

```diff
@@ -3,4 +3,7 @@
     "omnisharp_auto_start": true,
     "omnisharp_server_config_location":"",
     "omnisharp_onsave_codecheck": true,
-}
\ No newline at end of file
+    "omnisharp_onsave_showwarningwindows":true,
+    "omnisharp_onsave_showerrorwindows":true,
+    "omnisharp_debug_log":false

```

Comment:
@jchannon Ok.I make that's change for my selft use.I thought if I don't request merge last commit it's not request the commit for merge.